client/src/Organization/OrgSelect.tsx

```diff
@@ -53,6 +53,9 @@ export const OrgSelect = ({
                 }))();
     };
 
+    const handleInputLength = (inputValue: string) =>
+        inputValue.length <= 255 ? inputValue : inputValue.substr(0, 255);
```

Comment:
I don't really like the idea of just taking a substring without telling the user. Since we don't have great error handling / error presentation, can you add a `console.error("Event name too long")`? Thanks